purchase_stock_secondary_unit/tests/test_purchase_stock_secondary_unit.py

```diff
@@ -49,49 +50,49 @@ def test_confirm_purchase_order_without_secondary_uom(self):
                 line.secondary_uom_qty = 0.0
         self.purchase_order.button_confirm()
         picking = self.purchase_order.picking_ids
-        self.assertEqual(picking.move_lines.secondary_uom_qty, 0.0)
-        self.assertFalse(picking.move_lines.secondary_uom_id)
-        self.assertEqual(picking.move_lines.product_uom_qty, 10.0)
+        self.assertEqual(picking.move_line_ids.secondary_uom_qty, 0.0)
```

Comment:
Why are you changing model operations (stock.move) by model detailed operations (stock.move.line)

The same applies to several of the changes in this file.